tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -616,7 +619,7 @@ Status StorageManager::array_vacuum_fragments(
         "Cannot vacuum fragments; Array name cannot be null"));
 
   // Get all URIs in the array directory
-  URI array_uri(array_name);
+  URI array_uri = URI(array_name);
```

Comment:
make_uri () removal, but you're still not back to the original one action, think this constructs and then assigns, whereas the original approach just constructs... here and elsewhere.

Reply:
changed assignments back to constructors